projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/commet-set-header.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { ComponentFixture, inject, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { ToolbarEventService } from '../../../toolbar/toolbar.module';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import {CommentSetHeaderComponent} from './comment-set-header.component';
+
+describe('CommentSetHeader', () => {
+  let component: CommentSetHeaderComponent;
+  let fixture: ComponentFixture<CommentSetHeaderComponent>;
+  let nativeElement;
+  let toolbarEventService: ToolbarEventService
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
```

Comment:
I'm not sure we need this, as we're not testing any api calls

Reply:
good spot